### ISSUE TITLE
Issue in merging several vcfs

### ISSUE THREAD

**Issue by sekhwal** [samtools/bcftools#1419]
Hi,
I installed bcftools and I ran bgzip and indexed the files using following commands. 
bgzip -c file1.vcf > file1.vcf.gz
bcftools index file1.vcf.gz

However, once I merged using following command, terminal shows "killed". I have total 100 vcfs.
bcftools merge `ls *.vcf.gz` -O v > merged.vcf

Please advise how to overcome.

**Comment by pd3** [comment]
The program seems killed by your system. Check what version of bcftools are you using. Does it work with fewer files?

By the way, you don't have to do the `` `ls *vcf.gz` `` expansion, this can be run simply as `bcftools merge *.vcf.gz`. Unless you are using some obscure shell, that is.

**Comment by sekhwal** [comment]
Yes! it works with 5 or less than 5 vcfs. However, once I do with more than 9 vcfs. The process is Killed.

I created an conda environment  and I installed bcftools using the command: conda install bcftools. if I use bcftools --version, it shows:

bcftools 1.11
Using htslib 1.11
Copyright (C) 2020 Genome Research Ltd.

Please let me know if it is correct.

**Comment by valeriuo** [comment]
What does the kernel log show you?
Run `cat /var/log/kern.log`, if you have permission, or `dmesg -k` just after `bcftools` is killed.
